recipes/vtk/all/conanfile.py

```diff
@@ -0,0 +1,1468 @@
+            "smp_enable_OpenMP":       [True, False],
+            "smp_enable_TBB":          [True, False],
+
+            ### Modules ###
+            "build_all_modules":       [True, False],     # The big one, build everything - good for pre-built CCI
+
+            "enable_remote_modules":   [True, False],   # Build modules that exist in Remote folder
+                                                        # Currently LookingGlass, DICOM and MomentInvariants filter
+                                                        # The default will be off, they should be built externally in the conan world (when available).
+
+            # Qt-specific modules
+            "qt_version": ["Auto", "5", "6"],
+
+            # Note: there are a LOT of "module_enable_XXX" and "group_enable_XXX" options loaded in init()
+        }
+
+
+    default_options = {
+            "shared": False,
+            "fPIC":   True,
+
+            ### Helpful option when hacking on VTK - faster to extract on linux too ###
+            "use_source_from_git": False, # False = use the tarball
+
+            ### Conan package choices ###
+            "with_jpeg":        "libjpeg",  # the current standard in other packages on CCI
+
+            ### Debugging / future proofing ###
+            "legacy_remove":    False,
+            "legacy_silent":    False,
+            "use_future_const": False,
+            "debug_leaks":      False,
+
+            "debug_module":     False,
+
+            ### Compile options ###
+            "use_64bit_ids":   "Auto",
+            "enable_kits":     False,
+            "enable_logging":  False,
+
+            ### Wrapping support ###
+            "enable_wrapping": False,
+            "wrap_java":       False,
+            "wrap_python":     False,
+            "use_tk":          False,
+
+            ### Python specifics ###
+            "build_pyi_files": False,
+
+            ### Advanced tech ###
+            "use_cuda":    False,
+            "use_memkind": False,
+            "use_mpi":     False,
+
+            ### SMP ###
+            "smp_implementation_type": "Sequential",
+            "smp_enable_Sequential":   False,
+            "smp_enable_STDThread":    False,
+            "smp_enable_OpenMP":       False,
+            "smp_enable_TBB":          False,
+
+            ### Modules ###
+            "build_all_modules":       True,    # disable to pick+choose modules
+
+            "enable_remote_modules":   False,   # see notes above
+
+            # Qt-specific modules
+            "qt_version":              "Auto",  # Use 6 if provided, else use 5
+        }
+
+
+    @property
+    def _this_version_module_json_filename(self):
+        return os.path.join(self.recipe_folder, f"modules-{self.version}.json")
+
+
+    def _override_options_values(self, new_values):
+        overridden = { opt : ["DEFAULT", "YES", "NO", "WANT", "DONT_WANT"] for opt in new_values }
+        self.options.update(overridden, new_values)
```

Comment:
This only takes one argument that is a dict

Reply:
Thanks!  So just change it to `self.options.update(overridden)` ?

Reply:
If you mean `self.options.update()`,
the signature is `conans/model/options.py:345` :
    `def update(self, options=None, options_values=None):`